pwa-tenant-config-generator/SECURITY.md

```
@@ -30,11 +30,17 @@ when the following packages have been updated:
 
 ### tsx > 4.23.0
 
-- tar ^7.5.16
+- tar ^7.5.21
+
+### eslint > 9.39.4
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)